gent_base.theme

```diff
@@ -1008,8 +1041,11 @@ function gent_base_preprocess_vesta(&$variables) {
   /* @var $entity Drupal\dg_vesta\Entity\Vesta */
   $entity = $variables['elements']['#vesta'];
 
+  // Because vesta entities are language undefined we need to prefix the
+  // language code.
+  $locale = \Drupal::languageManager()->getCurrentLanguage()->getId();
   $variables['entity'] = $entity->getVestaValue();
-  $variables['url'] = $entity->toUrl();
+  $variables['url'] = '/' . $locale . $entity->toUrl()->toString();
```

Comment:
Should it be `/locale/entityUrl` or `/localeentityUrl`?

`sprintf('/%s/s%', $locale, $entity->toUrl()->toString());`